src/components/Card/Card.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Box } from '@chakra-ui/core';
+import styled from '@emotion/styled';
+
+import { ITheme } from '../theme.d';
+
+export enum CardElevations {
```

Comment:
```
type CardElevation = "none" | "s" | "m"
```

see for example this example: 
https://github.com/mui-org/material-ui/blob/master/packages/material-ui/src/Button/Button.d.ts#L18

Reply:
Both are valid options, 

Also, you can't iterate on `Type`s
https://github.com/heetch/flamingo/pull/158/files#diff-a78fa9cb2142c80dcfc7f47ee08fdaa3R32

Reply:
Just to save a line on storybook... I don't think it justifies using enum here, making the usage of the component too verbose.
If you really want, you could export a convenience const:
```
export CardElevations: CardElevation[] = ["none", "s","m"]
```

Reply:
It's not "just to save a line", it also ensures that a component's documentation is synced with its values
@heetch/frontend what do you guys think?

Reply:
I appologize if I underminded the usefulness of this. Either way, the solution suggested above should help you to cover that use case and a non-verbose usage.

Reply:
I agree it should be an `enum`, those are the only possible values it has, that's what an `enum` is for.